packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -550,10 +556,8 @@ export class TableLayout<T> extends ListLayout<T> {
           this.persistedIndices.set(layoutInfo.parentKey, indices);
         }
 
-        let index = collectionNode.index;
-        if (layoutInfo.parentKey === 'body') {
-          index -= this.collection.headerRows.length;
-        }
+        // TODO: perhaps fallback to collectionNode.index
```

Comment:
why do we still have a todo?

Reply:
Ah good catch, thought I got all of these. I'll get rid of the todo